src/bindgen/ir/enumeration.rs

```diff
@@ -648,6 +681,18 @@ impl Item for Enum {
     }
 
     fn add_dependencies(&self, library: &Library, out: &mut Dependencies) {
+        if let Some(tag) = self.tag.clone() {
+            let path = Path::new(tag);
+
+            // If there is an external tag enum, then add it as a dependency.
+            if let Some(items) = library.get_items(&path) {
```

Comment:
This looks a lot like what `Type::add_dependencies_ignoring_generics` does for a `Type::path`, can we factor this out into a common method in `Dependencies`? Something like `Dependencies::add_path(&mut self, &Library, &Path)` or so?